Frontend/src/Components/Notifications/Notifications.jsx

```diff
@@ -29,6 +30,18 @@ const Notifications = () => {
     return <Loading />;
   }
 
+  const handleNotificationClick = (notif) => {
+    if (notif.type === "FOLLOW") {
+      navigate(`/users/${notif.usernameTarget}`);
+    } else if (
+      notif.type === "LIKE" ||
```

Comment:
nit: i would create a variable for this above line 33
```
const isListingNotif = notif.type === "LIKE" || notif.type === "PURCHASE || notif.type === "LIKE_PURCHASE"
```
so that you can use it here and on line 61

also is it possible to use the back-end enums here for the notif type?